index.html

```diff
@@ -4,9 +4,51 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style/styles.css">
-    <title>Document</title>
+    <title>Card Flipping Game</title>
 </head>
 <body>
+    <div class="container">
+        <h1>Welcom to Ms-Card Flipping Game</h1>
+        <div class="stop-watch">
+            <h2>00:00:00</h2>
+        </div>
+        <button class="start" id="start"><h2>Click to start game</h2></button>
+        <div class="game-board hidden" id="game-board">
+            <!-- all the Card will be added here dynamicaly from e js-->
+            <!--<div class="card">
```

Comment:
always, always remove dummy code. If it's still needed you should have a comment as to why commented out code is still needed